server/apps/pa_img/pa_img_datalayer.py

```diff
@@ -119,15 +144,13 @@ def find(self, resource, req, lookup):
                         fields['photos'] = 'true'
 
         if not fields:
-            fields['days_since'] = 1
-
-        size = int(req.get('size', '100')) if int(req.get('size', '100')) > 0 else 100
-        if size and size < 100:
-            fields['limit'] = size
+            url += '/latest'
+        else:
+            url += '/search'
 
-        page = int(req.get('from', '0')) // size + 1
-        if page > 1:
-            fields['page'] = page
+        offset, limit = int(req.get('from', '0')), max(10, int(req.get('size', '25')))
+        fields['limit'] = limit
+        fields['page'] = math.ceil(offset / limit) + 1  # starts with 1
```

Comment:
Here some items can be skipped.  For example for offset 29 and limit 5, the result is from page 6 limit 5 that will lbe from 30 to 35 instead of 29 to 34.